test/rules/assertions/requireParam.js

```diff
@@ -1008,10 +1171,10 @@ export default {
        * Assign the project to an employee.
        *
        * @param {object} employee - The employee who is responsible for the project.
-       * @param {string} employee.name - The name of the employee.
-       * @param {string} employee.department - The employee's department.
+       * @param {string} employee.name1 - The name of the employee.
+       * @param {string} employee.department1 - The employee's department.
        */
-      function assign({name, department}) {
+      function assign({name1, department1}) {
```

Comment:
And these above...

Reply:
This was accidentally leftover from debugging, so I've removed it.